packages/sanity/src/core/releases/tool/overview/ReleasesOverviewColumnDefs.tsx

```diff
@@ -22,9 +23,18 @@ import {type TableRelease} from './ReleasesOverview'
 
 const ReleaseTime = ({release}: {release: TableRelease}) => {
   const {t} = useTranslation()
+  const {timeZone, utcToCurrentZoneDate} = useTimeZone()
+  const {abbreviation: localeTimeZoneAbbreviation} = getLocalTimeZone()
+
   const {metadata} = release
 
-  const getTimeString = () => {
+  const getTimezoneAbbreviation = useCallback(() => {
+    if (timeZone.abbreviation === localeTimeZoneAbbreviation) return ''
+
+    return `(${timeZone.abbreviation})`
```

Comment:
super small nit but you can always make this be

```suggestion
return timeZone.abbreviation === localeTimeZoneAbbreviation ? '' : `(${timeZone.abbreviation})`
``` 

right?

Reply:
So much nicer, thank you!